modules/stochastic_tools/include/distributions/NormalDistribution.h

```diff
@@ -0,0 +1,31 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+
+#ifndef NORMALDISTRIBUTION_H
+#define NORMALDISTRIBUTION_H
+
+#include "BoostDistribution.h"
+
+class NormalDistribution;
+
+template <>
+InputParameters validParams<NormalDistribution>();
+
+/**
+ * A class used to generate Normal distribution via Boost
+ */
+#ifdef LIBMESH_HAVE_EXTERNAL_BOOST
+class NormalDistribution : public BoostDistribution<boost::math::normal_distribution<Real>>
```

Comment:
@aeslaughter - It would be awesome if we didn't have to put these CPP guards around every derived class. This might be possible is we list dummy types in the BoostDistributions base class so that those symbols are defined when BOOST is not available.

Reply:
Are you talking about?

```
#ifndef LIBMESH_HAVE_EXTERNAL_BOOST
typedef dummy_type_here boost::math::normal_distribution // this would need to be done in namespaces, but you get the idea.
typedef dummy_type_here boost...distribution // etc...
#endif 
```

Rather than do this, we should probably just implement them all. Also, with doing this the distributions will still compile and be visible, but not work (mooseError of some sort), which I don't like.

Reply:
I think we should alias these boost distributions to a set of dummy classes and avoid all of the CPP directives.

Reply:
Oh, I didn't see your comment before I posted but yeah, that's along the line I was thinking except "typedef is so C++03"... 😉

Reply:
You can merge this as is if you want. Just want to try to catch this before we get directives spread all over the place.